src/components/DefaultLayout/DefaultLayout.js

```diff
@@ -10,7 +10,7 @@ import Header from '../Header';
 import Footer from '../Footer';
 import Brands from '../Brands';
 import ShippingInfo from '../ShippingInfo';
-import Notifications from '../shared/Notifications';
+import { NotificationsContainer } from '@/shared';
```

Comment:
import should be without _Container_ in the name! Check export name to be just: **Notifications**